nativescript-angular/package.json

```diff
@@ -25,28 +40,38 @@
   },
   "dependencies": {
     "nativescript-intl": "~0.0.8",
+    "reflect-metadata": "~0.1.8"
+  },
+  "peerDependencies": {
     "@angular/core": "~4.0.0",
     "@angular/common": "~4.0.0",
     "@angular/compiler": "~4.0.0",
-    "@angular/http": "~4.0.0",
     "@angular/platform-browser": "~4.0.0",
-    "@angular/platform-browser-dynamic": "~4.0.0",
-    "@angular/forms": "~4.0.0",
+    "@angular/animations": "~4.0.0",
     "@angular/router": "~4.0.0",
+    "@angular/forms": "~4.0.0",
+    "@angular/http": "~4.0.0",
+    "tns-core-modules": "^3.0.0 || ^3.0.0-rc.1",
     "rxjs": "^5.0.1",
-    "reflect-metadata": "~0.1.8",
-    "punycode": "1.3.2",
-    "querystring": "0.2.0",
-    "url": "0.10.3"
+    "zone.js": "^0.8.4"
   },
   "devDependencies": {
-    "@angular/animations": "~4.0.0",
+    "@angular/core": "~4.0.0",
+    "@angular/common": "~4.0.0",
+    "@angular/compiler": "~4.0.0",
     "@angular/compiler-cli": "~4.0.0",
+    "@angular/platform-browser": "~4.0.0",
+    "@angular/animations": "~4.0.0",
+    "@angular/router": "~4.0.0",
+    "@angular/forms": "~4.0.0",
+    "@angular/http": "~4.0.0",
     "codelyzer": "~3.0.0-beta.4",
+    "nativescript-intl": "^0.0.8",
     "tns-core-modules": "internal-preview",
     "tslint": "~4.5.0",
     "typescript": "~2.2.1",
-    "zone.js": "^0.8.2"
-  },
-  "nativescript": {}
+    "rxjs": "^5.0.1",
+    "reflect-metadata": "^0.1.8",
```

Comment:
What is the correct one:
dependency - "reflect-metadata": "~0.1.8"
devDependency - "reflect-metadata": "^0.1.8"